### ISSUE TITLE
Code intel Accessibility - Auditing and remediating user journeys

### ISSUE THREAD

**Issue by macraig** [sourcegraph/sourcegraph#33685]
## Problem to solve
Code intel team-specific effort to achieve https://github.com/sourcegraph/sourcegraph/issues/31475 in preparation for Q3 audit.

## Measure of success
User journeys have been audited per the guidance in the artifacts below.

## Solution summary
- Auditing user journeys within Code Intel.

Artifacts:
- [How to conduct an accessibility audit](https://docs.sourcegraph.com/dev/background-information/web/accessibility)

## What specific customers are we iterating on the problem and solution with?
- https://github.com/sourcegraph/accounts/issues/278

## Impact on use cases
This work contributes to the overall effort to scale our business.

## Delivery plan
- [x] https://github.com/sourcegraph/sourcegraph/issues/33506 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33507 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33508 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33509 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33510 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33511 
- [ ] https://github.com/sourcegraph/sourcegraph/issues/33512 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33513 
- [ ] https://github.com/sourcegraph/sourcegraph/issues/33514 
- [ ] https://github.com/sourcegraph/sourcegraph/issues/33515 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33516 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33517 
- [x] https://github.com/sourcegraph/sourcegraph/issues/33518

**Comment by jyavorska** [comment]
@macraig double checking, this is only an audit but not remediation of anything that is found? If it does include fixes, it should be L3 rather than engineering milestone.

**Comment by macraig** [comment]
@jyavorska as far as I'm aware it should include fixes. That said given the amount of user journeys the team needs to audit we could scope it to only include audit. cc @taylorsperry who might have better insight on the requirements for Q2

**Comment by taylorsperry** [comment]
Thanks, @jyavorska and @macraig. María, you're correct: We're aiming to have user journeys audited *and* fixed by the start of Q3 (the OKR is [here](https://docs.google.com/document/d/1Wiup6SVeABjOpAF70b0h1zC44fwUgT0xeoqPbHWAn6k/edit#bookmark=id.tirq1ajdfa65)). The goal is to only have to do the comprehensive audit once because it's an expensive and lengthy process.

**Comment by jyavorska** [comment]
Ok thanks, I'll rename the title to make it clear it's not just auditing.

**Comment by macraig** [comment]
Closing as all audits and outstanding issues have been fixed. Remaining smaller issues will be handled by the Frontend team
